### DISCUSSION TITLE
How do I get to the sandbox created when i import from GitHub?

### DISCUSSION THREAD

**Discussion by VAggrippino** [codesandbox/codesandbox-client#6657] [Q&A]
## How do I get to the code imported from GitHub that stays up to date with my latest commits?

I'm relatively new to CodeSandbox and I'd like to start using it more. I imported a project from GitHub and it was automatically forked.

I want to make changes to my code locally, push them to the repo on GitHub, and share the CodeSandbox link to my code and demo, but I can't easily get the link to the page that automatically updates when I update the repository.

For example, https://github.com/VAggrippino/text-gradient-and-fill

I imported it to CodeSandbox, but the button said *Import and Fork*. I didn't intend to fork it, but it brought me to a page with a generated URL that didn't update when I updated the repository. I deleted that...

Web searches transpired...

So I learned that the only CodeSandbox URLs that'll be updated when I make commits have `/s/github` in them and I deduced that it will always be `/s/github/vaggrippino/<project-name>`, but I don't see anywhere to get that URL.

My home page / dashboard shows a *template* for the project I imported and two other *Recently Used Templates* (which I *haven't* used) and everything else says:

> Uh oh, you haven't created any sandboxes in this folder yet!
> Start with a &lt;a>template&lt;/a>

Just to be clear, that's *My Drafts*, *All Sandboxes*, *Templates*, *Repositories*, and *Recently Modified*.

The item on the Home page has a three-dot menu with the following options.

- Fork Template
- Open Template
- Open Template in New Tab
- Copy Template Link
- Export Template

None of those options open the `/s/github` page, place the link to it in my clipboard, or open or link to a page that contains the latest changes from GitHub.

Here's the functionality I was looking for:

> The imported repository will always stay up to date with your latest commits. This means that any change to the GitHub repository will be reflected immediately in the sandbox.
> -- https://codesandbox.io/docs/importing#import-from-github

Is there any way to take advantage of this functionality without just figuring out the right URL and manually typing it?

Thank you.

Update:
This is what I get when I click on *Open Template* or *Open Template in New Tab*. In the following screenshot, CodeSandbox is the window on the left and GitHub is the window on the right.
![image](https://user-images.githubusercontent.com/195491/165001361-7fbca252-a921-470a-b9a0-68e017386453.png)

**Comment by JamesACS** [comment]
Hey @VAggrippino 

If you go to this link: https://codesandbox.io/s/

And import your project, this should solve the issue!
